ui/store/actions.js

```diff
@@ -3273,6 +3273,20 @@ export function setRecoveryPhraseReminderLastShown(lastShown) {
   };
 }
 
+export function setOutdatedBrowserWarningLastShown(lastShown) {
+  return () => {
+    callBackgroundMethod(
```

Comment:
We should use the async `submitRequestToBackground` here instead. This `callBackgroundMethod` function should be marked as deprecated. We've been in the process of migrating away from using the callback-style background calls for years now.

Reply:
Also, this is an async call so it should be an `async` function that waits for the call to complete. That makes it possible for the caller to wait, if they want to. Otherwise there's no way to tell when the request has completed.